src/reducers/versions.tsx

```diff
@@ -977,7 +999,7 @@ export const fetchDiff = ({
     );
     // Set the current version to the newer one (the head vesion) so that
     // components can track its loading progress.
-    dispatch(actions.setCurrentVersionId({ versionId: headVersionId }));
+    //     dispatch(actions.setCurrentVersionId({ versionId: headVersionId }));
```

Comment:
Hmm. This was used to let the UI know that a version was loading. It might not break anything but we should probably keep it as is for now.